### DISCUSSION TITLE
Welcome to vanessa-runner Discussions!

### DISCUSSION THREAD

**Discussion by artbear** [silverbulleters/vanessa-runner#416] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lo0p3r** [comment]
Правильно ли я понимаю, что нет команды, которая бы загружала конфигурацию сразу из исходников и нужно делать в два шага?
1. Собрать cf из исходников с помощью `compile`
2. Загрузить полученный cf с помощью `load`

При этом для расширения можно как пойти таким же путём с помощью команд `compileexttocfe` и `loadext`, но можно сделать всё одной командой `compileext` .

Вроде и хочется, чтобы стало красиво, добавив команду `compiletocf`, которая бы только выгружала конфигурацию в cf, и изменив поведение `compile`, чтобы она сразу загружала конфигурацию из исходников, но это же сломает всем сборочные линии...
Планируется ли что-то сделать с этим?
